### DISCUSSION TITLE
Is there a 12 or 24 hour time format display setting?

### DISCUSSION THREAD

**Discussion by MikeTheFid** [code4recovery/12-step-meeting-list#1219] [General]
TSML is showing time in 24 hour format in both the list and details pages. I can't find any setting to change it back to 12 hour.

I have set the WordPress setting to 12 hour but it doesn't seem to affect TSML.

**Comment by lovethecodelabs** [comment]
try setting it in the php.ini or equivalent.

**Reply by lovethecodelabs** [reply]
[Date]
date.timezone = America/Chicago

**Reply by lovethecodelabs** [reply]
here's another thought: try saving wordpress permalinks and/or clearing cache if needed.

**Reply by MikeTheFid** [reply]
Thanks for your reply. The time zone isn't the issue, it is correct in TSML. 

It's the time format. It's displaying, for example, 19:30 instead of 7:30 pm.

**Reply by lovethecodelabs** [reply]
![time-and-date-settings](https://github.com/code4recovery/12-step-meeting-list/assets/685087/f74337c2-fa09-4827-8d68-9b91597b68e8)

**Reply by lovethecodelabs** [reply]
?

**Reply by MikeTheFid** [reply]
As I mentioned in the OP, I have the WordPress setting as shown in the image you posted. 

It appears TSML isn't using or conforming itself to that setting.

**Comment by joshreisner** [comment]
could you share a link?

**Reply by MikeTheFid** [reply]
https://centralwest2district3aa.org/meetings/

**Reply by joshreisner** [reply]
thanks. i'm seeing it as a 12h time - TSML UI uses the browser's locale setting to format the time in the user's preferred format. my browser language is set to `en-us` which is why i see it in us time format. `en-ca` also uses 12 hour time, but `en-gb` i know uses 24 hour.

what's your browser language set to?

<img width="850" alt="Screenshot 2023-09-19 at 7 09 24 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/2c890b58-a432-4b26-8328-1bb2866a859f">

**Reply by MikeTheFid** [reply]
That was the issue. THANK YOU!

Chrome was set to English (UK). Annoying.

**Comment by MikeTheFid** [comment]
![CW2D3MeetingsList](https://github.com/code4recovery/12-step-meeting-list/assets/8200912/94052c9a-8096-4c49-85ac-a2f1e50cd339)

**Reply by MikeTheFid** [reply]
What's odd, is that this issue isn't happening in a mobile Chrome browser, only desktop.

AH! It only seems to happen on Chrome. The time format is correct on Opera and Edge.

**Comment by MikeTheFid** [comment]
The issue was my Chrome language setting. The language was English (UK) and not English (Canada).

Thanks everyone for your help!
